airbyte-integrations/connectors/source-orbit/source_orbit/streams.py

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC
+from typing import Any, Iterable, Mapping, MutableMapping, Optional, cast
+
+import requests
+import urllib.parse
+
+from airbyte_cdk.sources.streams.http import HttpStream
+
+
+class OrbitStream(HttpStream, ABC):
+    url_base = "https://app.orbit.love/api/v1/"
+
+    def __init__(self, workspace: str, **kwargs):
+        super().__init__(**kwargs)
+        self.workspace = workspace
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        return None
+
+    def request_params(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> MutableMapping[str, Any]:
+        return {}
+
+    def parse_response(
+        self,
+        response: requests.Response,
+        stream_state: Mapping[str, Any] = None,
+        stream_slice: Mapping[str, Any] = None,
+        next_page_token: Mapping[str, Any] = None,
+    ) -> Iterable[Mapping]:
+        yield response.json()
```

Comment:
```suggestion
    def parse_response(
        self,
        response: requests.Response,
        stream_state: Mapping[str, Any] = None,
        stream_slice: Mapping[str, Any] = None,
        next_page_token: Mapping[str, Any] = None,
    ) -> Iterable[Mapping]:
        data = response.json()
        records = data['data']
        yield records
```
Please alter the stream schemas to unnest the records from the data array.